aspnet/mvc/views/layout.rst

```diff
@@ -0,0 +1,111 @@
+Layout
+=======
+By `Steve Smith`_
+
+Views frequently share visual and programmatic elements. In this article, you'll see how to specify one or more common layouts for your ASP.NET app, and how your views can share directives and run common code before rendering. 
```

Comment:
Last sentence: `In this article, you'll learn how to use common layouts, share directives and run common code before rendering in your ASP.NET app.`